install.sh

```diff
@@ -16,21 +16,21 @@ amd_lshw() {
   lshw -c display -numeric -disable network | grep -q 'vendor: .* \[1002\]'
 }
 
-gpu_check() {
-  if available lspci && lspci -d '10de:' | grep -q 'NVIDIA'; then
-    nvidia_available="true"
-  elif available lshw && nvidia_lshw; then
-    nvidia_available="true"
-  elif available nvidia-smi; then
-    nvidia_available="true"
-  fi
+# gpu_check() {
```

Comment:
Why are you commenting this out?

Reply:
I never completed the GPU checking stuff... Trying to get the design correct first before bringing on the maintenance of all the GPU backends

And because I started running shellcheck on everything, shellcheck was correctly identifying this stuff as unused shell scripts (correctly)...

Maybe I should just delete this stuff...

Reply:
I am fine with commenting it out.